docs/pages/repo/docs/reference/command-line-reference.mdx

```diff
@@ -178,6 +178,27 @@ Task details include:
 - `dependencies`: Tasks that must run before this task
 - `dependents`: Tasks that must be run after this task
 
+#### `--experimental-env-mode`
+
+<Callout type="info">
+  **Warning**: This in experimental flag, so its name and behavior can change.
+</Callout>
+
+`type: string`
+
+Controls the available variables to your tasks based upon the values specified in the `globalEnvPassthrough` and `envPassthrough` fields in `turbo.json`.
```

Comment:
Here and elsewhere on this page `globalEnvPassthrough` -> `experimentalGlobalEnvPassthrough`, etc.